vendor/github.com/gorilla/rpc/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012 Rodrigo Moraes. All rights reserved.
```

Comment:
Thanks for the fast review! I updated the code.

If there is a need for others to use similar functions (RPC/JSON calls) we could move this into some util package.